### ISSUE TITLE
Any way to use this on Android?

### ISSUE THREAD

**Issue by Nikoichu** [prettierlichess/prettierlichess#186]
I don't have high hopes, but is there any way to use these themes on the official app or the webapp version (if using firefox?) on Android?

**Comment by kieferro** [comment]
Thanks for the question. It definitely won't be possible in the official app, because prettierlichess is a browser extension and thus not compatible with it.
For Firefox Mobile, however, the chances are a bit better. As far as I know, it is already possible to use extensions on mobile, but only with Firefox Nightly or Firefox Beta. For the official version this is not supported yet, although Firefox is actually [working on it and wants to make it available soon](https://blog.mozilla.org/addons/2023/09/27/test-firefox-android-extensions-and-help-developers-prepare-for-an-open-mobile-ecosystem-in-december/).
However, before we can publish the extension for any of the firefox versions on android, we first need to check if the extension works properly on mobile at all and if we need to migrate some things. The extension is not designed for mobile devices yet and it is also a bit more difficult to test on mobile compared to desktop.
So the support for it is definitely planned, but until the official support from Firefox is released and the extension is adapted accordingly, it will probably still take a few months.

**Comment by BloodRaven0** [comment]
I installed the addon on Firefox beta as you suggested and installed Lichess as a webapp, and it works wonderfully!
The webapp is a bit different from the native app naturally, and performance is not the same, but it's so much prettier!

I'll play around with the addon and try to install some user themes. I'll report back any issues I encounter. Thank you! ❤️

**Comment by kieferro** [comment]
> I installed the addon on Firefox beta as you suggested and installed Lichess as a webapp, and it works wonderfully!

Great. I'm going to leave this open so I remember to look into it again when Firefox makes it available in the official version as well.

**Comment by BloodRaven0** [comment]
@kieferro  It used to work well, but recently there has been a bug with the top navigation bar, it obscures the top part of the board.

**Comment by kieferro** [comment]
Thanks for reporting. I've opened a separate issue for this problem (#206)

**Comment by da5nsy** [comment]
The mobile app only allows people to choose from a set of colors, not to choose arbitrary colors, which I think would be a great improvement
